src/js/utils/Select.jsx

```diff
@@ -276,7 +276,10 @@ class Select extends Component {
             isClearable={clearable}
             title=""
             delimiter={delimiter}
-            onMenuClose={multi ? () => this.sortOptionsByChecked(options, value) : null}
+            onMenuClose={() => {
+              if (multi) this.sortOptionsByChecked(options, value);
+              if (this.props.onMenuClose) this.props.onMenuClose();
```

Comment:
what is that change for?

Reply:
I needed to perform some kind of an action _onMenuClose_ and currently Select component didn't have an option to pass my own callback for it to run on menu close.
This doesn't change anything except runs a `props.onMenuClose` if it is passed